### ISSUE TITLE
Form elements need labels

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1634]
Each form element need a label. Several form elements are not labeled. For example:
• The ‘Destroy in X Months’ and ‘Add a password’ fields when creating a new file.
• Within the Share>Link menu the toggle buttons are read as blank.
• Within the Share>Embed menu the field with the HTML to embed the document is not labeled. It is also a textarea element, a
code element would be better suited for this.
• The ‘Months’ select element when creating a new file.
